internal/swagger/swagger.go

```diff
@@ -19,6 +19,10 @@ import (
 	"github.com/cyverse/QMS/internal/model"
 )
 
+// Note: the comments in this package don't conform to the convention of including the name of the entity that the
+// comment describes. The reason for this is because the comments appear as-is in the API documentation. Confusing
+// documentation is produced when the structure names appear in the API documentation.
```

Comment:
I noticed that type names were added to comments in this file in several places. The convention is actually deliberately being broken here because the type names were appearing in the Swagger docs, producing confusing documentation. I added this comment to indicate why the convention is being broken here.